client/my-sites/plans/current-plan/style.scss

```diff
@@ -138,7 +136,7 @@
 	margin-left: 16px;
 	min-width: 140px;
 
-	@include breakpoint-deprecated( ">660px" ) {
+	@include breakpoint-deprecated(">660px") {
```

Comment:
The last one.

Reply:
After refactoring and extracting the new pieces to new components I ended up not even having to touch this file 😄